### DISCUSSION TITLE
Thank you for eqMac!

### DISCUSSION THREAD

**Discussion by randcraw** [bitgapp/eqMac#803] [General]
I just wanted to express how much I appreciate eqMac.  The ability to equalize music on my mac is a huge game changer for me, since the music source apps that I use (like SiriusXM) don't allow me to customize EQ.  Without eqMac, listening to music on a mac was so much less engaging than it is now.

And thanks for the lifetime subscription option (purchase).  It's worth every penny.

**Comment by nodeful** [comment]
Thank you @randcraw 
I'm really happy to hear some good feedback every once in a while as most communication I get back from customers is feedback regarding some issues, which is also understandable.
Thank you for you support, the app wouldn't be here today without people like you!
Year 2023 should be big for eqMac as many new features and improvements are in the pipeline!

Kind Regards,
Roman - Founder of eqMac
